openshift-hack/images/Dockerfile.openshift

```diff
@@ -1,9 +1,11 @@
 # Build vpcctl binary
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 as vpc-builder
 WORKDIR /build
-RUN git clone https://github.com/openshift/cloud-provider-vpc-controller && \
-    cd cloud-provider-vpc-controller && \
-    make build-linux
+COPY vendor/cloud.ibm.com/cloud-provider-vpc-controller .
+COPY openshift-hack/images/go.mod .
```

Comment:
Why do we need this hacked go.mod? I don't think that's required when I've seen this hack before, eg for controller-gen

Reply:
At least because we need to replace cloud.ibm.com/cloud-provider-vpc-controller
❯ go get github.com/IBM-Cloud/cloud-provider-vpc-controller
go: downloading github.com/IBM-Cloud/cloud-provider-vpc-controller v0.0.0-20210902000028-15ad6c21b84c
go get: github.com/IBM-Cloud/cloud-provider-vpc-controller@none updating to
        github.com/IBM-Cloud/cloud-provider-vpc-controller@v0.0.0-20210902000028-15ad6c21b84c: parsing go.mod:
        module declares its path as: cloud.ibm.com/cloud-provider-vpc-controller
                but was required as: github.com/IBM-Cloud/cloud-provider-vpc-controller
❯ go get cloud.ibm.com/cloud-provider-vpc-controller
go get: unrecognized import path "cloud.ibm.com/cloud-provider-vpc-controller": reading https://cloud.ibm.com/cloud-provider-vpc-controller?go-get=1: 401 Unauthorized

Reply:
Then we need to make sure dependencies versions are the same as in https://github.com/IBM-Cloud/cloud-provider-vpc-controller/blob/opensource/go.mod
If we take the latest version of dependencies, build process fails